### DISCUSSION TITLE
Status "Fahrzeug wird nicht weiter geladen Laden erlaubt"

### DISCUSSION THREAD

**Discussion by JoeSchwettmann** [goecharger/go-eCharger-API-v2#206] [Q&A]
Wo finde ich in der API den Status "Fahrzeug wird nicht weiter geladen Laden erlaubt". Ich möchte erkennen, wenn die Wallbox nicht weiter lädt, weil das Auto es stoppt (z.B. wegen 80% Akku Schonung).

**Comment by 0xFEEDC0DE64** [comment]
wenn alw=true ist aber car!=Charging(2), dann wäre das der Fall.

meistens geht der car state dann auf WaitCar(3) oder Complete(4), wenn das Auto ganz abgesteckt ist, geht er auf Idle(1)

**Reply by JoeSchwettmann** [reply]
Ok, jetzt habe ich endlich mein Logik Problem erkannt: Wenn ich das Laden stoppe, weil nicht genügend PV vorhanden ist, geht er auch in diesen Zustand. Es ist jetzt schwierig zu erkennen, ob das Laden durch mein Stoppen, oder durch das Auto selber angehalten wurde. Da ich 2 Wallboxen habe, solle immer WB1 zuerst voll geladen werden. Ich könnte auch den SoC vom Auto berücksichtigen, nur ist dieser nicht sehr zuverlässig. Oder gib es noch einen Parameter der mir sagt, ob ich es zuletzt gestoppt habe, oder ähnliches. Oder könnte ich auch ohne den go-e Controller zu besitzen das Überschussladen über andere API Befehle steuern.

**Reply by 0xFEEDC0DE64** [reply]
es gibt eine andere diskussion hier auf github, wo ich mehr oder weniger gut den api key "ids" beschrieben habe, leider habe ich noch keine zeit gehabt für diesen eine offizielle doku zu schreiben (such nach überschuss in die firmware einbauen oder so)
